unblob/file_utils.py

```diff
@@ -429,80 +453,71 @@ def _fs_path(self, path: Path) -> _FSPath:
     def _ensure_parent_dir(self, path: Path):
         path.parent.mkdir(parents=True, exist_ok=True)
 
-    def _get_extraction_path(
-        self, path: Path, path_use_description: str
-    ) -> Optional[Path]:
+    def _get_extraction_path(self, path: Path, path_use_description: str) -> Path:
         fs_path = self._fs_path(path)
-        if fs_path.is_safe:
-            return fs_path.absolute_path
 
-        report = ExtractionProblem(
-            path=fs_path.format_path(),
-            problem=f"Potential path traversal through {path_use_description}",
-            resolution="Skipped.",
-        )
-        self.record_problem(report)
-        return None
+        if not fs_path.is_safe:
+            report = ExtractionProblem(
+                path=fs_path.format_path(),
+                problem=f"Potential path traversal through {path_use_description}",
```

Comment:
I would include in the report the path pointing to lost+found so this way we can get the original path easily

Reply:
done.